sites/website/src/docs/resources/cheat-sheet.md

```diff
@@ -0,0 +1,695 @@
+---
+id: cheat-sheet
+title: Cheat Sheet
+sidebar_label: Cheat Sheet
+custom_edit_url: https://github.com/microsoft/fast/edit/master/sites/website/src/docs/resources/cheat-sheet.md
+---
+
+# Cheat Sheet
+
+## Packages:
+
+### [@microsoft/fast-element](https://www.fast.design/docs/fast-element/getting-started)
+
+**A library that is a lightweight means to easily build performant, memory-efficient, standards-compliant Web Components.**
+
+* Provides a thin layer of opinions on top of [Web Components](https://www.fast.design/docs/resources/why-web-components), lifting the level of abstraction just enough to make it easier and FASTer to [build components]((#building-components)).   
+* Use this library when you want to create new custom web components. 
+
+To install the fast-element library, use either `npm` or `yarn`:
+
+```shell
+npm install --save @microsoft/fast-element
+```
+
+```shell
+yarn add @microsoft/fast-element
+```
+
+Within your JavaScript or TypeScript code, you can then import library APIs like this:
+
+```ts
+import { FASTElement } from "@microsoft/fast-element";
+```
+
+---
+
+### [@microsoft/fast-foundation](https://www.fast.design/docs/introduction)
+
+**A library that provides foundational building blocks that can be assembled to create new design systems and component libraries.**
+
+* Exports of this package can generally be thought of as un-styled base components that implement semantic and accessible markup and behavior.   
+* Use this library when you want to implement something like Google's Material Design or Twitter Bootstrap.
+
+```ts
+import {
+  Button as FoundationButton,
+  buttonTemplate as template,
+} from "@microsoft/fast-foundation";
+import { css } from "@microsoft/fast-element";
+
+export class MyButton extends FoundationButton {
+  ...
+}
+
+export const buttonStyles = css`
+  :host {
+    background-color: azure;
+  }
+`;
+
+export const myButton = MyButton.compose({
+  baseName: "button",
+  baseClass: FoundationButton,
+  template,
+  styles: buttonStyles,
+  shadowOptions: {
+    delegatesFocus: true,
+  },
+});
+```
+
+---
+
+### [@microsoft/fast-components](https://www.fast.design/docs/components/getting-started)     
+
+**A component library that implements Microsoft's [FAST Frame Design System](https://www.fast.design/docs/design-systems/fast-frame/).**
+
+* Assembles the building blocks of `@microsoft/fast-foundation` to create its component set. 
+* Use this library when you want to integrate [FAST Components](#using-components) into an existing site or app.
+
+To install the components, use either `npm` or `yarn`:
+
+```shell
+npm install --save @microsoft/fast-components
+``` 
+
+```shell
+yarn add @microsoft/fast-components
+```
+
+To use a Web Component as a custom element in HTML, the custom element must be registered:
```

Comment:
This seems generic but we're talking specifically about the library in this case. Perhaps instead of "To use a Web Component" here we can be more specific to the package?

```suggestion
To use a component as a custom element in HTML, the custom element must be registered.
```

Reply:
@chrisdholt Thank you - updated!